packages/terra-menu/src/terra-dev-site/doc/example/ArrowMenu.jsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import Button from 'terra-button';
+import PropTypes from 'prop-types';
+import Menu from 'terra-menu';
+import classNames from 'classnames/bind';
+import styles from './BasicMenu.module.scss';
+
+const cx = classNames.bind(styles);
```

Comment:
Can you explain why this was added? Is this replacing the usage of 
`<BasicMenu isArrowDisplayed />` seen [here](https://github.com/cerner/terra-framework/pull/1056/files#diff-f54a0ae78bc54090261bdc8f031abe0bL36)?

Reply:
Passing `isArrowDisplayed` as prop in mdx file was not working as expected so, I made a new example file for it.